documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -1,15 +1,11 @@
 # Build and Deploy SAPUI5 or SAP Fiori Applications on SAP Cloud Platform with Jenkins
 
-Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Cloud Foundry environment.
 
 ## Prerequisites
 
-* You have installed the Java Runtime Environment 8.
```

Comment:
I personally think we should rely on docker from that personal point of view the cleanup here is OK. BUT: there was never a corresponding decision by product management / product owner (these roles are not that clearly defined in this project ...). So we should still be able to run on a plain Jenkins with all the tools installed locally. Hence we should still document that.

Reply:
the central documentation isn't updated yet. i removed most prerequisites and pointed to the piper guided tour.